internal/testutil/assert/assert.go

```diff
@@ -23,6 +25,7 @@ var errorCompareFn = func(e1, e2 error) bool {
 	return e1.Error() == e2.Error()
 }
 var errorCompareOpts = cmp.Options{cmp.Comparer(errorCompareFn)}
+var ErrCallbackTimedOut = errors.New("callback timed out")
```

Comment:
Will `lint` fail here because there's no comment?

Reply:
Surprisingly, no. Probably because it's a variable and not a function

Reply:
IIRC happens for variables too. Maybe the `internal` package is treated differently, which makes sense. Either way, new commit has the comment so I'm resolving this.